### PR TITLE
fix `Images with style=background-image: url("img.png"); aren't loaded` (close #1212)

### LINKED ISSUE #1212: Images with style = background-image: url("img.png"); aren't loaded

### PR DESCRIPTION
**Pull request by LavrovArtem**
/cc @miherlosev @helen-dikareva

### CODE REVIEW

File: test/client/fixtures/sandbox/node/window-test.js [hunk 1]
```diff
@@ -375,3 +375,15 @@ test('the constructor field of a function should return a wrapped Function objec
     Function.prototype.toString = nativeToString;
     /*eslint-enable no-extend-native*/
 });
+
+test('getPropertyValue and setProperty methods of css object should be overridden (GH-1212)', function () {
+    var div      = document.createElement('div');
+    var url      = 'http://some.domain.com/image.png';
+    var proxyUrl = urlUtils.getProxyUrl(url);
+
+    div.style.setProperty('background', 'url(' + url + ')', '');
```
Comment (miherlosev) [inline test/client/fixtures/sandbox/node/window-test.js]: Add test for non-string url value case
For example:
 `div.style.setProperty('background', null)`

File: test/client/fixtures/sandbox/node/window-test.js [hunk 2]
```diff
@@ -375,3 +375,15 @@ test('the constructor field of a function should return a wrapped Function objec
     Function.prototype.toString = nativeToString;
     /*eslint-enable no-extend-native*/
 });
+
+test('getPropertyValue and setProperty methods of css object should be overridden (GH-1212)', function () {
```
Comment (miherlosev) [inline test/client/fixtures/sandbox/node/window-test.js]: Also check `style.removeProperty`.
https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration/removeProperty

It should returns non-proxied value.

**Comment by LavrovArtem** [comment]
FPR

**Comment by LavrovArtem** [comment]
ping @helen-dikareva

**Pull request merged**